scripts/replace-cli-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+VERSIONS=$(gh api -H "Accept: application/vnd.github+json" /repos/github/codeql-cli-binaries/releases | jq '.[].tag_name' | head -2)
+
+# we are exporting these variables so that we can access these variables in the workflow
+export LATEST_VERSION=$(echo $VERSIONS | awk '{ print $1 }' | sed "s/\"//g")
+export PREVIOUS_VERSION=$(echo $VERSIONS | awk '{ print $2 }' | sed "s/\"//g")
+
```

Comment:
You need to export these variables for later steps:

```suggestion
echo "LATEST_VERSION=$LATEST_VERSION" >> $GITHUB_ENV
echo "PREVIOUS_VERSION=$PREVIOUS_VERSION" >> $GITHUB_ENV
```